domains/cats.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "yunexiz",
+    "discord": "1040303561847881729",
+    "email": ""
+  },
+  "record": {
+    "CNAME": "cats.is-a.dev"
```

Comment:
Uh you cant put CNAME as the sane subdomain you are trying to get

Reply:
oops